### ISSUE TITLE
[1.15.2] ava.lang.NullPointerException: Initializing game 	at net.minecraft.client.renderer.color.BlockColors.func_186722_a(BlockColors.java:97) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}

### ISSUE THREAD

**Issue by SDUBZ** [thraaawn/BonsaiTrees#159]
Forge: 31.1.16-18
BonsaiTrees-2.1.2.6

logs:
[latest.log](https://github.com/thraaawn/BonsaiTrees/files/4290847/latest.log)

crash report:
[crash-2020-03-05_14.35.40-client.txt](https://github.com/thraaawn/BonsaiTrees/files/4290850/crash-2020-03-05_14.35.40-client.txt)

**Comment by davenonymous** [comment]
Not a Bonsai Trees issue. Your actual error is this:
```
[14:35:33] [modloading-worker-6/ERROR]: Failed to create mod instance. ModID: mysticalagriculture, class com.blakebr0.mysticalagriculture.MysticalAgriculture
java.lang.NullPointerException: null
	at com.github.elrol.industrialagriculture.init.SoulInit.addSoul(SoulInit.java:126) ~[?:1.15.2-1.6.0]
	at com.github.elrol.industrialagriculture.init.SoulInit.init(SoulInit.java:60) ~[?:1.15.2-1.6.0]
	at com.github.elrol.industrialagriculture.plugin.MAPlugin.onRegisterMobSoulTypes(MAPlugin.java:17) ~[?:1.15.2-1.6.0]
	at com.blakebr0.mysticalagriculture.registry.PluginRegistry.lambda$loadPlugins$2(PluginRegistry.java:41) ~[?:3.0.0]
	at com.blakebr0.mysticalagriculture.registry.PluginRegistry$$Lambda$4167/361570988.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1249) ~[?:1.8.0_51]
	at com.blakebr0.mysticalagriculture.registry.PluginRegistry.loadPlugins(PluginRegistry.java:41) ~[?:3.0.0]
	at com.blakebr0.mysticalagriculture.MysticalAgriculture.<init>(MysticalAgriculture.java:71) ~[?:3.0.0]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_51]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[?:1.8.0_51]
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_51]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:131) [?:31.1]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer$$Lambda$2211/1627145296.accept(Unknown Source) [?:31.1]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) [?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$2210/905962412.accept(Unknown Source) [?:1.8.0_51]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) [?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$2210/905962412.accept(Unknown Source) [?:1.8.0_51]
	at net.minecraftforge.fml.ModContainer.transitionState(ModContainer.java:112) [?:?]
	at net.minecraftforge.fml.ModList.lambda$null$10(ModList.java:134) [?:?]
	at net.minecraftforge.fml.ModList$$Lambda$2590/1224314740.accept(Unknown Source) [?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) [?:1.8.0_51]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) [?:1.8.0_51]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512) [?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291) [?:1.8.0_51]
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.execLocalTasks(ForkJoinPool.java:1040) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1058) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) [?:1.8.0_51]
```
It's an industrial agriculture error. Please report the issue there.

**Comment by skellitor301** [comment]
I've had this issue as well and I don't have Industrial Agriculture.

> ---- Minecraft Crash Report ----
> // I bet Cylons wouldn't have this problem.
> 
> Time: 4/15/20 7:58 PM
> Description: Initializing game
> 
> java.lang.NullPointerException: Initializing game
> 	at net.minecraft.client.renderer.color.BlockColors.func_186722_a(BlockColors.java:97) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
> 	at com.davenonymous.bonsaitrees2.misc.PotColorizer.init(PotColorizer.java:33) ~[?:2.0.2.35] {re:classloading}
> 	at net.minecraftforge.eventbus.ASMEventHandler_280_PotColorizer_init_Block.invoke(.dynamic) ~[?:?] {}
> 	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:80) ~[eventbus-1.0.0-service.jar:?] {}
> 	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:258) ~[eventbus-1.0.0-service.jar:?] {}
> 	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:168) ~[?:28.2] {re:classloading}
> 	at net.minecraftforge.fml.ModLoader.lambda$postEvent$30(ModLoader.java:243) ~[?:?] {re:classloading}
> 	at net.minecraftforge.fml.ModLoader$$Lambda$5214/2089561347.accept(Unknown Source) ~[?:?] {}
> 	at java.util.HashMap.forEach(HashMap.java:1280) ~[?:1.8.0_51] {}
> 	at net.minecraftforge.fml.ModList.forEachModContainer(ModList.java:217) ~[?:?] {re:classloading}
> 	at net.minecraftforge.fml.ModLoader.postEvent(ModLoader.java:243) ~[?:?] {re:classloading}
> 	at net.minecraftforge.client.ForgeHooksClient.onBlockColorsInit(ForgeHooksClient.java:193) ~[?:?] {re:classloading}
> 	at net.minecraft.client.renderer.color.BlockColors.func_186723_a(BlockColors.java:76) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
> 	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:502) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
> 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:365) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
> 	at net.minecraft.client.main.Main.main(SourceFile:155) [?:?] {re:classloading}
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
> 	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
> 	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:56) [forge-1.14.4-28.2.4.jar:28.2] {}
> 	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$411/430976509.call(Unknown Source) [forge-1.14.4-28.2.4.jar:28.2] {}
> 	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-4.1.0.jar:?] {}
> 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-4.1.0.jar:?] {}
> 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-4.1.0.jar:?] {}
> 	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-4.1.0.jar:?] {}
> 	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-4.1.0.jar:?] {}
> 
> 
> A detailed walkthrough of the error, its code path and all known details is as follows:
> ---------------------------------------------------------------------------------------
> 
> -- Head --
> Thread: Client thread
> Stacktrace:
> 	at net.minecraft.client.renderer.color.BlockColors.func_186722_a(BlockColors.java:97)
> 	at com.davenonymous.bonsaitrees2.misc.PotColorizer.init(PotColorizer.java:33)
> 	at net.minecraftforge.eventbus.ASMEventHandler_280_PotColorizer_init_Block.invoke(.dynamic)
> 	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:80)
> 	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:258)
> 	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:168)
> 	at net.minecraftforge.fml.ModLoader.lambda$postEvent$30(ModLoader.java:243)
> 	at net.minecraftforge.fml.ModLoader$$Lambda$5214/2089561347.accept(Unknown Source)
> 	at java.util.HashMap.forEach(HashMap.java:1280)
> 	at net.minecraftforge.fml.ModList.forEachModContainer(ModList.java:217)
> 	at net.minecraftforge.fml.ModLoader.postEvent(ModLoader.java:243)
> 	at net.minecraftforge.client.ForgeHooksClient.onBlockColorsInit(ForgeHooksClient.java:193)
> 	at net.minecraft.client.renderer.color.BlockColors.func_186723_a(BlockColors.java:76)
> 	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:502)
> 
> -- Initialization --
> Details:
> Stacktrace:
> 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:365)
> 	at net.minecraft.client.main.Main.main(SourceFile:155)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:497)
> 	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:56)
> 	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$411/430976509.call(Unknown Source)
> 	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
> 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
> 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
> 	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81)
> 	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65)
> 
> -- System Details --
> Details:
> 	Minecraft Version: 1.14.4
> 	Minecraft Version ID: 1.14.4
> 	Operating System: Windows 10 (amd64) version 10.0
> 	Java Version: 1.8.0_51, Oracle Corporation
> 	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
> 	Memory: 2392694280 bytes (2281 MB) / 3642753024 bytes (3474 MB) up to 7635730432 bytes (7282 MB)
> 	CPUs: 4
> 	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8192m -Xms256m -XX:PermSize=256m
> 	ModLauncher: 4.1.0+62+5bfa59b
> 	ModLauncher launch target: fmlclient
> 	ModLauncher naming: srg
> 	ModLauncher services: 
> 		/eventbus-1.0.0-service.jar eventbus PLUGINSERVICE 
> 		/forge-1.14.4-28.2.4.jar object_holder_definalize PLUGINSERVICE 
> 		/forge-1.14.4-28.2.4.jar runtime_enum_extender PLUGINSERVICE 
> 		/accesstransformers-1.0.5-shadowed.jar accesstransformer PLUGINSERVICE 
> 		/forge-1.14.4-28.2.4.jar capability_inject_definalize PLUGINSERVICE 
> 		/forge-1.14.4-28.2.4.jar runtimedistcleaner PLUGINSERVICE 
> 		/forge-1.14.4-28.2.4.jar fml TRANSFORMATIONSERVICE 
> 	FML: 28.2
> 	Forge: net.minecraftforge:28.2.4
> 	FML Language Providers: 
> 		javafml@28.2
> 		minecraft@1
> 		kotlinfml@1.4.0
> 	Mod List: 
> 		NetherPortalFix_1.14.4-7.0.0.jar NetherPortalFix {netherportalfix@7.0.0 CREATE_REGISTRIES}
> 		mcw-windows-1.0.1-mc1.14.4.jar Macaw's Windows {mcwwindows@1.0.1 CREATE_REGISTRIES}
> 		KleeSlabs_1.14.4-7.4.8.jar KleeSlabs {kleeslabs@7.4.8 CREATE_REGISTRIES}
> 		rsgauges-1.14.4-1.2.2.jar Gauges and Switches {rsgauges@1.2.2 CREATE_REGISTRIES}
> 		Neat 1.5-19.jar Neat {neat@1.5-19 CREATE_REGISTRIES}
> 		ForgeEndertech-1.14.4-5.2.2.0-build.0082.jar Forge Endertech {forgeendertech@5.2.2.0 CREATE_REGISTRIES}
> 		xnet-1.14-1.8.7-alpha.jar XNet {xnet@1.14-1.8.7-alpha CREATE_REGISTRIES}
> 		Controlling-5.0.7.jar Controlling {controlling@5.0.6 CREATE_REGISTRIES}
> 		Placebo-1.14.4-2.3.5.jar Placebo {placebo@2.3.5 CREATE_REGISTRIES}
> 		mightyarchitect-mc1.14.4_v0.5.jar The Mighty Architect {mightyarchitect@0.5 CREATE_REGISTRIES}
> 		WitherSkeletonTweaks-1.14.4-3.1.0.jar Wither Skeleton Tweaks {wstweaks@3.1.0 CREATE_REGISTRIES}
> 		Bookshelf-1.14.4-4.4.71.jar Bookshelf {bookshelf@4.4.71 CREATE_REGISTRIES}
> 		unstabletools-3.6b.jar Unstable Tools {unstabletools@3.6b CREATE_REGISTRIES}
> 		Waddles-1.14.4-0.8.3.jar Waddles {waddles@1.14.4-0.8.3 CREATE_REGISTRIES}
> 		SnowRealMagic-1.14.4-1.5.2.jar Snow! Real Magic! {snowrealmagic@1.5.2 CREATE_REGISTRIES}
> 		JustEnoughResources-1.14.4-0.10.1.76.jar Just Enough Resources {jeresources@0.10.1.76 CREATE_REGISTRIES}
> 		shetiphiancore-1.14-3.6.1.jar ShetiPhian-Core {shetiphiancore@3.6.1 CREATE_REGISTRIES}
> 		mysticallib-1.14.4-1.16.1.jar Mystical Lib {mysticallib@1.14.4-1.16.1 CREATE_REGISTRIES}
> 		refinedstorage-1.7.3.jar Refined Storage {refinedstorage@1.7.3 CREATE_REGISTRIES}
> 		TreeGrowingSimulator2020-1.0.0-beta+30.jar Tree Growing Simulator 2020 {treegrowingsimulator@1.0.0-beta+30 CREATE_REGISTRIES}
> 		AdPoles-1.14.4-3.0.1.2-build.0072.jar To the Bat Poles! {adpoles@3.0.1.2 CREATE_REGISTRIES}
> 		mining_dimension-1.14.4-1.0.10.jar Mining World {mining_dimension@1.14.4-1.0.10 CREATE_REGISTRIES}
> 		simplylight-0.9.0.jar Simply Light {simplylight@0.9.0 CREATE_REGISTRIES}
> 		ElecCore-1.14.4-2.0.30-Beta.jar ElecCore {eleccore@2.0.30 CREATE_REGISTRIES, eleccoreloader@2.0.30 CREATE_REGISTRIES}
> 		curios-FORGE-1.14.4-1.0.6.1.jar Curios API {curios@FORGE-1.14.4-1.0.6.1 CREATE_REGISTRIES}
> 		tetra-1.14.4-1.27.0.jar Tetra {tetra@1.14.4-1.27.0 CREATE_REGISTRIES}
> 		Patchouli-1.14.4-1.1-26.jar Patchouli {patchouli@1.14.4-1.1-25 CREATE_REGISTRIES}
> 		morecharcoal-1.14.4-6.jar More Charcoal {morecharcoal@1.14.4-6 CREATE_REGISTRIES}
> 		AutoRegLib-1.4-35.jar AutoRegLib {arl@1.4-35 CREATE_REGISTRIES}
> 		enchxchg-1.14.4-5.0.0.jar Enchantments Exchanger {enchxchg@1.14.4-5.0.0 CREATE_REGISTRIES}
> 		OreExcavation-1.6.147.jar Ore Excavation {oreexcavation@NONE CREATE_REGISTRIES}
> 		libnonymous-1.14.4-1.0.4.25.jar Libnonymous {libnonymous@1.0.4.25 CREATE_REGISTRIES}
> 		tombstone-4.3.3-1.14.4.jar Corail Tombstone {tombstone@4.3.3 CREATE_REGISTRIES}
> 		SpartanWeaponry-1.14.4-2.0.0-alpha-3.jar Spartan Weaponry {spartanweaponry@2.0.0 alpha 3 CREATE_REGISTRIES}
> 		mcw-roofs-1.0.2-mc1.14.4.jar Macaw's Roofs {mcwroofs@1.0.2 CREATE_REGISTRIES}
> 		ImmersiveRailroading-1.7.3_1.14.4.jar Immersive Railroading {immersiverailroading@1.7.3 CREATE_REGISTRIES, universalmodcore@1.0 ERROR}
> 		AI-Improvements-1.14.4-0.3.0.jar AI-Improvements {aiimprovements@0.2.2 CREATE_REGISTRIES}
> 		moreoverlays-1.16.1.jar More Overlays {moreoverlays@1.16.1 CREATE_REGISTRIES}
> 		mcw-furniture-1.0.1-mc1.14.4.jar Macaw's Furnitures {mcwfurnitures@1.0.1 CREATE_REGISTRIES}
> 		Huds-1.14.4-1.1.7.jar Huds {huds@version CREATE_REGISTRIES}
> 		PrettyBeaches_1.14.4-3.0.7.jar Pretty Beaches {prettybeaches@3.0.7 CREATE_REGISTRIES}
> 		StevesFactoryManager-1.14.4-3.0.0-alpha26+hotfix.jar Steve's Factory Manager {sfm@1.14.4-3.0.0-alpha CREATE_REGISTRIES}
> 		BetterAdvancements-1.14.4-0.1.0.90.jar Better Advancements {betteradvancements@0.1.0.90 CREATE_REGISTRIES}
> 		Kiwi-1.14.4-2.6.3.jar Kiwi {kiwi@2.6.3 CREATE_REGISTRIES}
> 		SoundControl-1.14.4-0.1.0.0.jar §3Dynamic Surroundings: §eSound Control {sndctrl@1.14.4-0.1.0.0 CREATE_REGISTRIES}
> 		torchslabmod-1.14.4_v1.5.jar Torch Slab Mod {torchslabmod@1.5 CREATE_REGISTRIES}
> 		pamhc2trees-1.14.4-1.0.0.jar Pam's HarvestCraft 2 Fruit Trees {pamhc2trees@version CREATE_REGISTRIES}
> 		craftingstation-2.1.6.jar Crafting Station {craftingstation@2.1.6 CREATE_REGISTRIES}
> 		snad-1.14.4-1.7.11.10a.jar Snad {snad@1.7.11.10a CREATE_REGISTRIES}
> 		Mine and Slash-1.14.4-2.9.4.jar Mine and Slash {mmorpg@2.8.0 CREATE_REGISTRIES}
> 		platforms-1.14-1.5.1.jar Platforms {platforms@1.5.0 CREATE_REGISTRIES}
> 		jei-1.14.4-6.0.0.27.jar Just Enough Items {jei@6.0.0.27 CREATE_REGISTRIES}
> 		JustEnoughCalculation-1.14.4-3.4.2.jar Just Enough Calculation {jecalculation@3.4.2 CREATE_REGISTRIES}
> 		caelus-FORGE-1.14.4-1.0.1.2.jar Caelus API {caelus@FORGE-1.14.4-1.0.1.2 CREATE_REGISTRIES}
> 		colytra-FORGE-1.14.4-2.0.0.1.jar Colytra {colytra@FORGE-1.14.4-2.0.0.1 CREATE_REGISTRIES}
> 		MS Auto-Compat-1.14.4-1.6.0.jar M&S Auto-Compat 1.14.4 {azurecompat@1.6.0 CREATE_REGISTRIES}
> 		create-mc1.14.4_v0.2.2b.jar Create {create@mc1.14-0.2.2b CREATE_REGISTRIES}
> 		Waystones_1.14.4-5.1.1.jar Waystones {waystones@5.1.1 CREATE_REGISTRIES}
> 		mgui-1.14.4-1.2.0.jar mgui {mgui@1.2.0 CREATE_REGISTRIES}
> 		journeymap-1.14.4-5.7.0b1.jar Journeymap {journeymap@1.14.4-5.7.0b1 CREATE_REGISTRIES}
> 		comforts-FORGE-1.14.4-2.0-beta6.jar Comforts {comforts@FORGE-1.14.4-2.0-beta6 CREATE_REGISTRIES}
> 		tumbleweed-1.14-0.4.7.jar Tumbleweed {tumbleweed@1.14-0.4.7 CREATE_REGISTRIES}
> 		good-nights-sleep-1.14.4-1.0.3.jar Good Night's Sleep {goodnightsleep@1.0.3 CREATE_REGISTRIES}
> 		SereneSeasons-1.14.4-2.0.0.40-universal.jar Serene Seasons {sereneseasons@1.14.4-2.0.0.40 CREATE_REGISTRIES}
> 		cursedearth-1.4a.jar Cursed Earth {cursedearth@1.4a CREATE_REGISTRIES}
> 		BotanyPots-1.14.4-1.0.10.jar BotanyPots {botanypots@1.0.10 CREATE_REGISTRIES}
> 		VanillaTweaks-1.14.4.1.5.12.jar VanillaTweaks {vanillatweaks@1.14.4.1.5.12 CREATE_REGISTRIES}
> 		AdHooks-1.14.4-4.0.0.2-build.0072.jar Advanced Hook Launchers {adhooks@4.0.0.2 CREATE_REGISTRIES}
> 		curioofundying-FORGE-1.14.4-2.0.1.jar Curio of Undying {curioofundying@FORGE-1.14.4-2.0.1 CREATE_REGISTRIES}
> 		StepUp-1.14.4-0.1.0.jar StepUp {stepup@1.14.4-0.1.0 CREATE_REGISTRIES}
> 		BonsaiTrees-2.0.2.35.jar Bonsai Trees 2 {bonsaitrees2@2.0.2.35 CREATE_REGISTRIES}
> 		alib-1.14.4-8.jar A Lib {alib@1.14.4-8 CREATE_REGISTRIES}
> 		mcjtylib-1.14-3.5.17-alpha.jar McJtyLib {mcjtylib@1.14-3.5.17-alpha CREATE_REGISTRIES}
> 		rftoolsbase-1.14-0.0.12-alpha.jar RFToolsBase {rftoolsbase@1.14-0.0.12-alpha CREATE_REGISTRIES}
> 		rftoolsstorage-1.14-0.0.7-alpha.jar RFToolsStorage {rftoolsstorage@1.14-0.0.7-alpha CREATE_REGISTRIES}
> 		walljump-1.14.4-1.3.1.jar Wall-Jump! {walljump@1.14.4-1.3.1 CREATE_REGISTRIES}
> 		ToastControl-1.14.4-2.2.0.jar Toast Control {toastcontrol@2.2.0 CREATE_REGISTRIES}
> 		immersive-portals-0.15(for mc 1.14 with forge).jar Immersive Portals {immersive_portals@0.15 CREATE_REGISTRIES}
> 		mysticalworld-1.14.4-1.9.0.jar Mystical World {mysticalworld@1.14.4-1.9.0 CREATE_REGISTRIES}
> 		extradisks-1.14.4-1.1.jar Extra Disks {extradisks@1.14.4-1.1 CREATE_REGISTRIES}
> 		firstaid-1.7.12.jar First Aid {firstaid@1.7.12 CREATE_REGISTRIES}
> 		forge-1.14.4-28.2.4-universal.jar Forge {forge@28.2.4 CREATE_REGISTRIES}
> 		MiniCoal2-1.14.4-0.0.1-universal.jar MiniCoal 2 {minicoal2@{{version}} CREATE_REGISTRIES}
> 		immersiveposts-1.14.4-1.0.0-rev1.jar Immersive Posts {immersiveposts@1.0.0-rev1 CREATE_REGISTRIES}
> 		ironchest-1.14.4-9.1.5.jar Iron Chests {ironchest@1.14.3-9.0.3 CREATE_REGISTRIES}
> 		SimpleDivingGear-3.0.4 mc 1.14.4.jar Simple Diving Gear {simpledivegear@3.0.4 mc 1.14.4 CREATE_REGISTRIES}
> 		CraftingTweaks_1.14.4-10.2.1.jar Crafting Tweaks {craftingtweaks@10.2.1 CREATE_REGISTRIES}
> 		alchemistry-1.14.4-11.jar Alchemistry {alchemistry@1.14.4-11 CREATE_REGISTRIES}
> 		forge-1.14.4-28.2.4-client.jar Minecraft {minecraft@1.14.4 CREATE_REGISTRIES}
> 		rftoolsutility-1.14-1.2.14-alpha.jar RFToolsUtility {rftoolsutility@1.14-1.2.14-alpha CREATE_REGISTRIES}
> 		Environs-1.14.4-0.1.0.0.jar §3Dynamic Surroundings: §eEnvirons {environs@1.14.4-0.1.0.0 CREATE_REGISTRIES}
> 		theoneprobe-1.14-1.4.37.jar The One Probe {theoneprobe@NONE CREATE_REGISTRIES}
> 		swingthroughgrass-1.14.4-1.3.2.jar SwingThroughGrass {swingthroughgrass@1.14.4-1.3.2 CREATE_REGISTRIES}
> 		MouseTweaks-2.13-mc1.14.4.jar Mouse Tweaks {mousetweaks@2.13 CREATE_REGISTRIES}
> 		ImmersiveEngineering-0.14-94.jar Immersive Engineering {immersiveengineering@0.14-94 CREATE_REGISTRIES}
> 		bluepower-1.14.4-0.3.183-alpha-universal.jar Blue Power {bluepower@0.3.183-alpha CREATE_REGISTRIES}
> 		pamhc2crops-1.14.4-1.0.4.jar Pam's HarvestCraft 2 Crops {pamhc2crops@version CREATE_REGISTRIES}
> 		astikorcarts-1.0.2-1.14.4.jar AstikorCarts {astikorcarts@1.0.2 CREATE_REGISTRIES}
> 		rftoolsbuilder-1.14-1.2.12-alpha.jar RFToolsBuilder {rftoolsbuilder@1.14-1.2.12-alpha CREATE_REGISTRIES}
> 		paintings-1.14.4-6.0.1.5.jar Paintings ++ {paintings@1.14.4-6.0.1.5 CREATE_REGISTRIES}
> 		MobEffects-1.14.4-0.1.0.0.jar §3Dynamic Surroundings: §eMob Effects {mobeffects@1.14.4-0.1.0.0 CREATE_REGISTRIES}
> 		SpartanShields-1.14.4-2.0.0.jar Spartan Shields {spartanshields@2.0.0 CREATE_REGISTRIES}
> 		Quark-r2.0-212.jar Quark {quark@r2.0-212 CREATE_REGISTRIES}
> 		pamhc2foodcore-1.14.4-1.0.3.jar Pam's HarvestCraft 2 Food Core {pamhc2foodcore@version CREATE_REGISTRIES}
> 		FTBUtilitiesBackups-2.0.0.11.jar FTB Utilities: Backups {ftbbackups@2.0.0.11 CREATE_REGISTRIES}
> 		chemlib-1.14.4-17.jar Chem Lib {chemlib@1.14.4-17 CREATE_REGISTRIES}
> 		StorageDrawers-1.14.4-6.0.7.jar Storage Drawers {storagedrawers@1.14.4-6.0.6 CREATE_REGISTRIES}
> 		DoubleSlabs-1.14-1.2.3.jar Double Slabs {doubleslabs@1.2.3 CREATE_REGISTRIES}
> 		Statues-1.14.4-0.1.3.2.jar Statues Mod {statues@0.1.3.2 CREATE_REGISTRIES}
> 		AppleSkin-mc1.14.4-forge-1.0.13.jar AppleSkin {appleskin@1.0.13 CREATE_REGISTRIES}
> 		engineersdecor-1.14.4-1.0.20-b4.jar Engineer's Decor {engineersdecor@1.0.20-b4 CREATE_REGISTRIES}
> 		Aquaculture-1.14.4-2.0.7.jar Aquaculture 2 {aquaculture@1.14.4-2.0.7 CREATE_REGISTRIES}
> 		betterbedrockgenerator-1.14.4-0.0.6.jar Better Bedrock Generator {betterbedrockgenerator@0.0.6 ERROR}
> 		CHAS v1.3 for 1.14.jar Craftable Horse Armour {craftablehorsearmour@1.3 CREATE_REGISTRIES}
> 		refinedstorageaddons-0.5.jar Refined Storage Addons {refinedstorageaddons@0.5 CREATE_REGISTRIES}
> 		BetterFoliage-2.5.0-Forge-1.14.4.jar Better Foliage {betterfoliage@2.5.0 CREATE_REGISTRIES}
> 	Launched Version: forge-28.2.4
> 	LWJGL: 3.2.2 build 10
> 	OpenGL: GeForce GTX 1070/PCIe/SSE2 GL version 4.6.0 NVIDIA 445.75, NVIDIA Corporation
> 	GL Caps: Using GL 1.3 multitexturing.
> Using GL 1.3 texture combiners.
> Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
> Shaders are available because OpenGL 2.1 is supported.
> VBOs are available because OpenGL 1.5 is supported.
> 
> 	Using VBOs: Yes
> 	Is Modded: Definitely; Client brand changed to 'forge'
> 	Type: Client (map_client.txt)
> 	Resource Packs: 
> 	Current Language: English (US)
> 	CPU: 4x Intel(R) Core(TM) i5-2400 CPU @ 3.10GHz
